scripts/ci/gitlab/pipeline/publish.yml

```diff
@@ -189,18 +189,18 @@ publish-draft-release:
   # the releases of one another. Use resource_group to ensure that at most one instance of this job
   # is running at any given time.
   resource_group:                  crates-publishing
-  variables:
-    # crates.io rate limits crates publishing by 1 per minute, so a delay needs to be inserted
-    # slightly higher than that after publishing each crate. The value is specified in seconds.
-    SPUB_AFTER_PUBLISH_DELAY: 64
```

Comment:
This delay is part of an implementation detail and thus it has been moved to the `publish-crates` script itself.